tests/test_clean.py

```diff
@@ -24,11 +24,11 @@ def test_only_text_is_cleaned():
 
     with pytest.raises(TypeError) as e:
         clean(some_type)
-    assert "argument cannot be of 'type' type" in str(e)
+    assert "argument cannot be of 'type' type" in str(e.value)
```

Comment:
The `requirements-dev.txt` file specifically uses `pytest<5`. Updating the code to handle recent versions of pytest should be done in a separate PR.

Reply:
Yeah, this predates that. So I think we should focus on the second two commits.

Reply:
Removed this commit after rebase from master.